testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1161,3 +1161,35 @@ def server_secret
   xpath = "//td[contains(text(), '#{action_name}')]/ancestor::tr[contains(@class, 'list-row-even')]/td/div/button/#{button}"
   raise "xpath: #{xpath} not found" unless find(:xpath, xpath).click
 end
+
+
+# todo is this the right place?
+# common constants for SSH bootstrap with SSH key auth test
+key_filename = 'id_rsa_bootstrap-passphrase_linux.pub'
+## authorized_keys paths on the client
+auth_keys_path = '/root/.ssh/authorized_keys'
+auth_keys_sav_path = '/root/.ssh/authorized_keys.sav'
+
```

Comment:
I would prefer to have it as part of constants.rb file. But we couldn't arrive at an agreement related to constants in our squad.

Reply:
I think these constants should be part of the individual step definitions:

1 - if they were real global constants, they should start with `$`
2 - they are too specific to become really global

Please move them into the steps. It's not a big deal if they are repeated a few times.

Reply:
Moving to steps.